CalibrationPoint[] points
int32 selected_point
float32[] transformation